framework/log/Target.php

```diff
@@ -342,4 +348,17 @@ public function getEnabled()
 
         return $this->_enabled;
     }
+
+    /**
+     * Returns formatted ('Y-m-d H:i:s') timestamp for message.
+     * If [[microtime]] is configured to true it will return format 'Y-m-d H:i:s.u'
+     * @param float $timestamp
+     * @return string
+     */
+    protected function getTime($timestamp)
+    {
+        list ($timestamp, $usec) = explode('.', (string)$timestamp);
+
+        return date('Y-m-d H:i:s', $timestamp) . ($this->microtime ? ('.' . $usec) : '');
```

Comment:
LGTM, but i'd write 
```php
$format = $this->microtime ? 'Y-m-d H:i:s.u' : 'Y-m-d H:i:s';
return date($format);
```

Reply:
Me too, but `date` will not work as expected:

"u - Note that date() will always generate 000000 since it takes an integer parameter"

http://php.net/manual/en/function.date.php